BUILDER_DEV.md

```diff
@@ -213,6 +213,17 @@ HTTP/1.1 201 Created
 }
 ```
 
+## Habitat Studio Based Development Environment
+This will start everything to run builder in a studio except the web app
+
+1. `hab studio enter` (This will download and install and start the builder service)
+1. `export HAB_AUTH_TOKEN=<my_token>` (Unless you have one set in your cli.toml)
+1. `sl` Wait until you see the origin service start
+1. `origin`
+1. `keys`
+1. `load_packages`
```

Comment:
If this relies on an actual set of packages somewhere, should we explain where they should go, and link to the archive somehow?